versions.json

```diff
@@ -1,19 +1,19 @@
 {
     "azcopy": {
```

Comment:
Possible to move everything under common distro?

Reply:
alma is different. In theory I could move the 2 ubuntus under common and have alma as an override, but then what happens when we add another distro like mariner? 

for this pass I had decided to just move things that were common across all to common just to simplify things and then look at more complicated patterns (or possibly more complicated logic) later